db/migrate/20180202093541_create_protocol_transfers.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class CreateProtocolTransfers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :protocol_transfers do |t|
+      t.references :from, references: :person, null: false
+      t.references :to, references: :person, null: false
+      t.references :protocol_subscription, foreign_key: true, null: false
+
+      t.timestamps
+    end
+    add_foreign_key :protocol_transfers, :people, column: :from_id
+    add_foreign_key :protocol_transfers, :people, column: :to_id
+
+    student1 = Person.find_by_mobile_phone('0637422582')
+    student2 = Person.find_by_mobile_phone('0638736269')
+
+    mentor_old = Person.find_by_mobile_phone('0613404142')
+    mentor_new = Person.find_by_mobile_phone('0637422582')
```

Comment:
Alleen even de vraag of we wel telefoonnummers in migrations willen opnemen van een publieke repo op github. Waarschijnlijk kan dit niet in een migration maar een of ander scriptje dat we moeten copypasten.